content/en/docs/components/central-dash/profiles.md

```diff
@@ -6,13 +6,17 @@ weight = 20
 
 ## What is a Kubeflow Profile?
 
-A Profile is a Kubernetes CRD introduced by Kubeflow that wraps a Kubernetes Namespace.
+A Kubeflow Profile is a [Kubernetes CRD](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/#customresourcedefinitions) introduced by Kubeflow that wraps a Kubernetes [Namespace](https://kubernetes.io/docs/tasks/administer-cluster/namespaces-walkthrough/).
+
```

Comment:
Can we keep this as a single paragraph, It's a bit short otherwise. 

That is, can we remove these new lines.

Reply:
![image](https://github.com/user-attachments/assets/4181a917-927e-4d14-a7e5-689c5cddb075)